### PR TITLE
libshout: Update to 2.4.3

### PR DESCRIPTION
**Pull request by neheb**
Removed inactive maintainer.

Rearranged Makefile for consistency between packages.

Added --without-pic to avoid both -fPIC and -fpic.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @thess 
Compile tested: powerpc8540

### CODE REVIEW

**Comment by neheb** [comment]
Just had a thought on the second patch. AFAIK, Tremor is being used as it uses less RAM. But nowadays with OpenWrt, the unofficial minimum is 8/64MB devices. Any thoughts on removing the patch?

**Comment by diizzyy** [comment]
floating point operations are extremely slow on hardware without a fpu like MIPS so no. Preferably it should only default to tremor on devices (platforms) that really need it.

File: libs/libshout/Makefile [hunk 1]
```diff
@@ -6,21 +6,21 @@
...
-PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(BUILD_VARIANT)/$(PKG_NAME)-$(PKG_VERSION)
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=http://downloads.us.xiph.org/releases/libshout/
-PKG_HASH:=f3acb8dec26f2dbf6df778888e0e429a4ce9378a9d461b02a7ccbf2991bbf24d
+PKG_SOURCE_URL:=https://downloads.us.xiph.org/releases/libshout/
+PKG_HASH:=0d8af55d1141bf90710bcd41a768c9cc5adb251502a0af1dd22c8da215d40dfe
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(BUILD_VARIANT)/$(PKG_NAME)-$(PKG_VERSION)
 
+PKG_MAINTAINER:=Ted Hess <thess@kitschensync.net>
 PKG_LICENSE:=LGPL-2.0+
```
Comment (cotequeiroz) [inline libs/libshout/Makefile]:
```suggestion
PKG_LICENSE:=LGPL-2.0-or-later
```
Might as well update the SPDX license tag.

**Comment by thess** [comment]
Tremor is used not for RAM savings, but performance on non-float systems. Please make sure Tremor is retained - I have never noticed any audio quality degradation from its use.

**Comment by neheb** [comment]
I've kept it. Just wondering.

**Pull request merged**